enlighten/_counter.py

```diff
@@ -676,7 +681,11 @@ def format(self, width=None, elapsed=None):
         # Otherwise return a counter
         fields['fill'] = u'{0}'
         rtn = self.counter_format.format(**fields)
-        return rtn.format(u' ' * (width - len(rtn) + self.offset + 3))
+        if self.offset is not None:
+            ret = rtn.format(u' ' * (width - len(rtn) + self.offset + 3))
+        else:
+            ret = rtn.format(u' ' * (width - self.manager.term.length(rtn) + 3))
+        return ret
```

Comment:
Why not just use something like this?
`barWidth = width - self.manager.term.length(rtn) + self.offset + 3  # 3 is for the bar placeholder`

Reply:
That would count the offset twice on older code (eg the old multicounter) as term.length would correct for the offset...

Reply:
Yes, but I see this as a breaking change, major version, and do not intend to get rid off offset.